doc/api/cli.md

```diff
@@ -94,6 +94,25 @@ Follows `require()`'s module resolution
 rules. `module` may be either a path to a file, or a node module name.
 
 
+### `--inspect[=host:port]`
+<!-- YAML
+added: v6.5.0
```

Comment:
I think `--inspect` was initially added in [6.3.0](https://nodejs.org/en/blog/release/v6.3.0/). It might have changed slightly since then, but luckily we have per item changelogs now.